presto-hive/src/main/java/com/facebook/presto/hive/BackgroundHiveSplitLoader.java

```diff
@@ -343,12 +341,12 @@ private static boolean shouldUseFileSplitsFromInputFormat(InputFormat<?, ?> inpu
         }
         catch (NestedDirectoryNotAllowedException e) {
             // Fail here to be on the safe side. This seems to be the same as what Hive does
-            throw new PrestoException(HIVE_INVALID_BUCKET_FILES, format("%s Found sub-directory in bucket directory for partition: %s", CORRUPT_BUCKETING, splitFactory.getPartitionName()));
+            throw new PrestoException(HIVE_INVALID_BUCKET_FILES, format("%s Found sub-directory in bucket directory for partition: %s", getCorruptBucketingMsg(), splitFactory.getPartitionName()));
         }
 
         // verify we found one file per bucket
         if (files.size() != bucketCount) {
-            throw new PrestoException(HIVE_INVALID_BUCKET_FILES, format("%s The number of files in the directory (%s) does not match the declared bucket count (%s) for partition: %s", CORRUPT_BUCKETING, files.size(),
+            throw new PrestoException(HIVE_INVALID_BUCKET_FILES, format("%s The number of files in the directory (%s) does not match the declared bucket count (%s) for partition: %s", getCorruptBucketingMsg(), files.size(),
```

Comment:
The argument wrapping is wrong. The style is to wrap either all arguments or none, so all arguments to `format()` should be wrapped (the existing code is wrong but we should fix it now since we are changing it)

Reply:
Yes I noticed it too. Already changed.